modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayBuildAdministrator.testcase

```diff
@@ -77,22 +82,29 @@ definition {
 
 		Testray.deleteAllProjects();
 
-		Testray.tearDownCP();
+		// Testray.tearDownCP();
 
-		if ("${testPortalInstance}" == "true") {
-			PortalInstances.tearDownCP();
-		}
+		// if ("${testPortalInstance}" == "true") {
+		// 	PortalInstances.tearDownCP();
+		// }
 	}
 
 	@description = "This is a test for LPS-156438 - Verify if as an Administrator can Add a Build"
 	@priority = "5"
 	test AdminCanAddBuild {
 		task ("Given that the user add a new build") {
+			ApplicationsMenu.gotoSite(site = "Testray 2");
+
+			Testray.changePagination(
+				numberItems = "20",
+				valuePagination = "150");
+
 			Testray.goToProjectPage(projectName = "Projects_1 Project");
```

Comment:
Hi @joelrocha68, I think these steps can be added in setup, as it is repeated in all tests